DynamoDBIndexingCore/Factories/AssetEntityFactory.cs

```diff
@@ -39,7 +39,7 @@ public static Asset ToDomainAsset(this Document databaseEntity)
                 AssetId = databaseEntity["assetId"],
                 AssetType = databaseEntity["assetType"],
                 AssetAddress = ((Document) databaseEntity["assetAddress"]).ToDomainAssetAddress(),
-                Tenure = databaseEntity["tenure"].GetType() != typeof(DynamoDBNull) ? ((Document) databaseEntity["tenure"]).ToDomainTenureForAsset() : null,
+                Tenure = databaseEntity.Contains("tenure") ? ((Document) databaseEntity["tenure"]).ToDomainTenureForAsset() : null,
```

Comment:
we need a double condition here as if it does contain the 'tenure' key, then we still need to check if it is of type 'DynamoDBNull'